doc/toc.yml

```diff
@@ -47,6 +47,9 @@
 - name: .NET MAUI Embedding
   href: Learn/Maui/toc.yml
   topicHref: xref:Uno.Extensions.Maui.Overview
+- name: Theme Service
+  href: Learn/ThemeService/toc.yml
+  topicHref: xref:Uno.Extensions.ThemeService.Overview
```

Comment:
@Kunal22shah we need to avoid having twice the "Theme Service" here please
![image](https://github.com/user-attachments/assets/857a76ff-2910-4ddb-8e44-8c9b88480981)

You can take a look at what was done for Samples for example 
![image](https://github.com/user-attachments/assets/5deb54b2-cba9-4d3f-8af2-57681c5f5c6e)